.github/workflows/build-product.yml

```diff
@@ -0,0 +1,62 @@
+name: Build APIM with tests
+
+on:
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+    branches:
+    - master
+  schedule:
+    - cron:  '0 20 * * *'
```

Comment:
can't  set the branch here?

Reply:
As per the doc "Scheduled workflows run on the latest commit on the default or base branch." which means the master branch.
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onschedule